### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jurekbarth**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "vi-css-helper": "git+https://github.com/virtualidentityag/vi-css-helper.git#1.1.1"
   },
   "devDependencies": {
-    "@biotope/build": "6.0.0",
+    "@biotope/build": "next",
```
Comment (SheepFromHeaven) [inline package.json]: Doesn't the next version need a whole different folder structure?

Comment (jurekbarth) [reply package.json]: next is the current beta, it's maybe not the best naming. I will change it to beta with our next release 👍

File: projectConfig.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 module.exports = {
 	project: 'Biotope - The Frontend Ecosystem Framework',
 	global: {
+		tasks: {
+			htmlhint: false
```
Comment (timomayer) [inline projectConfig.js]: Why false per default?

Comment (jurekbarth) [reply projectConfig.js]: together with biotope element and data passed as json it always throws errors (`The value of attribute [ your-attribute ] must be in double quotes.`), as it expects double quotes for attributes. We use single quotes to have double quotes for json. Therefor it's less useful in the biotope-element context.

**Pull request merged**